app/templates/content/buyers-guide.html

```diff
@@ -0,0 +1,189 @@
+<div class="inner-block floated-children">
+<div class="govspeak">
+
+<h2 id="buying-services-on-digital-marketplace">
+<span class="number">1. </span>Digital Marketplace overview</h2>
+<p>
+  The Digital Marketplace helps you find and compare cloud technology and people to deliver digital projects.
+</p>
+<p>
+  It uses 3 frameworks (agreements between government and suppliers).
+</p>
+<p>
+  Cloud services (eg web hosting or IT health checks) are on the G-Cloud framework.
+</p>
+<p>
+  Specialists who can work on digital projects (eg technical architects and user researchers) are on the Digital Services framework.
+</p>
+<p>
+  Physical datacentre space for legacy systems is available on the Crown Hosting Data Centres framework.
+</p>
+<p>
+  All public sector organisations, including agencies and arm’s length bodies, can use the Digital Marketplace.
+  <a href='https://www.gov.uk/government/publications/public-sector-organisations-eligible-to-use-cloudstore'>Check the list of approved organisations</a> to see if you’re eligible.
+</p><p>
+  All Digital Marketplace suppliers have successfully been through the Official Journal of the European Union (OJEU) process. That means you can save on the time and cost traditionally associated with entering into individual procurement contracts.
+</p>
+
+<h2 id="g-cloud-order-form">
+  <span class="number">2. </span>Digital Marketplace services</h2>
+<p>
+  There are over 19,000 services on the Digital Marketplace, split into 4 categories:
+</p>
+<ul>
+  <li>
+    Infrastructure as a Service (IaaS)
+  </li><li>
+    Platform as a Service (PaaS)
+  </li><li>
+    Software as a Service (SaaS)
+  </li><li>
+    Specialist Cloud Services (SCS)
+  </li>
+</ul>
+<p>
+More than 170 suppliers provide digital specialists to perform all or some of the following roles:
+</p>
+<table class='dss-table'>
+  <thead><tr>
+    <th>Capabilities</th>
+    <th>Linked roles</th>
+  </tr></thead>
+  <tbody>
+    <tr>
+      <td>Software engineering and ongoing support</td>
+      <td>Developer, technical architect</td>
+    </tr>
+    <tr>
+      <td>Product development and service design</td>
+      <td>Product manager, business analyst</td>
+    </tr>
+    <tr>
+      <td>Agile delivery management</td>
+      <td>Delivery manager, product manager, business analyst</td>
+    </tr>
+    <tr>
+      <td>Front-end design and interaction design</td>
+      <td>Designer</td>
+    </tr>
+    <tr>
+      <td>Content design and development</td>
+      <td>Content designer</td>
+    </tr>
+    <tr>
+      <td>System administration and web operations</td>
+      <td>Web operations, security specialist, database specialist, network specialist</td>
+    </tr>
+    <tr>
+      <td>User research</td>
+      <td>Researcher</td>
+    </tr>
+    <tr>
+      <td>Embedding Agile</td>
+      <td>Agile coach</td>
+    </tr>
+  </tbody>
+</table>
+<p>
+  1 supplier provides datacentre hosting to government. It offers:
```

Comment:
This isn't the latest version of the content - latest version has "One supplier" and a more recent updated date (still needs to be deployed to prod though, I thought it had): https://github.gds/gds/digitalmarketplace-frontend/blob/master/grails-app/views/digital-marketplace-buyers-guide.gsp